app/models/user.rb

```diff
@@ -47,11 +47,11 @@ def send_reset_confirmation
   end
 
   def first_identity
-    active_identities[0] unless active_identities.empty?
+    active_identities[0]
```

Comment:
Not according to the tests. `first_identity` is only called in the context of SLO, and a user will always have at least one active identity when SLO is initiated. The only exception is if the user is signed out and then attempts to log out. In that case, we can't call `user.first_identity` since there is no user. That is why we have `return NullIdentity.new unless user` in `SingleLogoutHandler#identity`.